Foundation/NSData.swift

```diff
@@ -217,7 +217,11 @@ open class NSData : NSObject, NSCopying, NSMutableCopying, NSSecureCoding {
     }
     
     open var bytes: UnsafeRawPointer {
-        return UnsafeRawPointer(CFDataGetBytePtr(_cfObject))
+        guard let bytes = CFDataGetBytePtr(_cfObject) else {
+            //This could occure on empty data being encoded.
+            return UnsafeRawPointer([])
+        }
+        return UnsafeRawPointer(bytes)
```

Comment:
I am claiming that the objective-c version is not correctly annotated. Try this in objc:

```objc
assert([NSData data].bytes == NULL);
```

The header for NSData should probably be marked as 

```objc
@property (readonly) const void *_Nullable bytes NS_RETURNS_INNER_POINTER;
```